backend/python/app/graphql/types/comment_type.py

```diff
@@ -0,0 +1,19 @@
+import graphene
+
+
+class CreateCommentDTO(graphene.InputObjectType):
+    story_translation_content_id = graphene.Int(required=True)
+    user_id = graphene.Int(required=True)
+    time = graphene.Int(required=True)
```

Comment:
can we instead insert time inside the story service? i.e. not allow the user to pass in the time